### DISCUSSION TITLE
Help on setup

### DISCUSSION THREAD

**Discussion by JamieP83** [wictorwilen/ring-timelapse#7] [Q&A]
Hey!
sorry to be a pain but im not sure where/how to start here. 
i run Home Assistant on a RPI4. 

ive got portainer running and gone to setup a new container, however im not sure what to select on this first screen! sorry!

![image](https://user-images.githubusercontent.com/48566948/154443241-02c9bbdd-10b8-4250-a164-9e74f62d7e89.png)

**Comment by elminster13** [comment]
that field is the image only so you should put "wictorwilen/ring-timelapse"
